src/Language/PureScript/Interactive/Types.hs

```diff
@@ -22,13 +49,33 @@ data PSCiConfig = PSCiConfig
 -- The let bindings are partial,
 -- because it makes more sense to apply the binding to the final evaluated expression.
 data PSCiState = PSCiState
-  { psciImportedModules     :: [ImportedModule]
-  , psciLetBindings         :: [P.Declaration]
-  , psciLoadedExterns       :: [(P.Module, P.ExternsFile)]
-  } deriving Show
+  [ImportedModule]            -- ^ psciImportedModules
+  [P.Declaration]             -- ^ psciLetBindings
```

Comment:
Haddock is crashing on this line. It doesn't seem like you're allowed to annotate the individual arguments to a constructor, just the whole constructor or record fields: https://www.haskell.org/haddock/doc/html/ch03s02.html

Reply:
Ah, thanks